doc/modules/clustering.rst

```diff
@@ -1388,3 +1388,30 @@ Drawbacks
   concepts of clusters, such as density based clusters like those obtained
   through DBSCAN.
 
+.. _contingency_matrix:
+
+Contingency Matrix
+----------------------
+
+Contingency matrix is a format displaying the multivariate frequency distribution of the variables. In a typical contingency matrix, rows are true labels, and columns are predicted labels, which can be different from true labels. Each cell represents the interaction of true and predicted labels.
+
```

Comment:
Also, saying predictions can be different from true labels is a bit ambiguous. Clearly they can be different, but they could live in an entirely different space. Maybe an example where labels_true is `` [0, 0, 0, 1, 1, 1]`` and labels_pred is ``["red", "red", "green", "green", "blue", "blue"]`` would be better?